ambry-store/src/main/java/com.github.ambry.store/StoreFindToken.java

```diff
@@ -182,21 +182,33 @@ static StoreFindToken fromBytes(DataInputStream stream, StoreKeyFactory factory)
           case JournalBased:
             // read incarnationId
             String incarnationId = Utils.readIntString(stream);
```

Comment:
`readIntString()` works fine for zero length strings, right? I am assuming the test verifies that.